doc/conf.py

```diff
@@ -287,6 +287,12 @@
         'sklearn': 'http://scikit-learn.org/stable'}
     }
 
+# Get rid of spurious warnings due to some interaction between
+# autosummary and numpydoc. See
+# https://github.com/phn/pytpm/issues/3#issuecomment-12133978 for more
+# details
+numpydoc_show_class_members = False
```

Comment:
This is the slightly magical bit of this PR. I am not sure what that does but it does remove warnings created by some interaction between numpydoc and autosummary. I checked and the reference documentation still looks fine.